### PR TITLE
Liqo Agent (un)installer

### PR DESCRIPTION
**Pull request by AbakusW**
# Description

This PR adds the possibility in the *Liqo* installer to also (un)install the **Liqo Agent** version contained in the latest *Liqo* release.

For this occasion, both the *Agent* internal and desktop banner icons have been renewed.

### Integration pipeline

* update of some workflow commands according to the new github actions syntax

* addition of the steps required to build the *Agent* binary and upload it in the newly created release when a tag is pushed

### Installer

* addition of the *Agent* installer that can be triggered by previously setting
```bash
INSTALL_AGENT=true
```
* the binary is directly downloaded from the latest Liqo release

* the *Agent* is installed as a desktop application by means of '.desktop' template files.

### Icons

* the *Agent* iconset has been updated according to *Liqo* official logo.

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -142,45 +141,70 @@ jobs:
...
+      - name: Send coverage
+        uses: shogo82148/actions-goveralls@v1
+        with:
+          path-to-profile: coverage.txt
+      - name: Build Agent asset
+        run: |
+          GO111MODULE=on CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build ./cmd/tray-agent/liqo-agent.go
+          tar -czf liqo-agent.tar.gz liqo-agent
+        if: github.event_name == 'push' && github.event.repository.full_name == 'liqotech/liqo' && startsWith(github.ref, 'refs/tags/v')
+      - name: Create Release
```
Comment (palexster) [inline .github/workflows/go.yml:193]: May be we can cache the artifact and retrieve it in the "Release" job, where the release should be created ?

https://docs.github.com/en/free-pro-team@latest/actions/guides/storing-workflow-data-as-artifacts

Comment (AbakusW) [reply .github/workflows/go.yml:193]: yes. I explored this possibility and chose to continue on the same workflow as there are just two additional steps, but we can move them.

Comment (palexster) [reply .github/workflows/go.yml:193]: Thanks

**Review by palexster — APPROVED** [review]

**Comment by palexster** [comment]
Superseded by #339,#338,#337

**Pull request closed without merging**